test/Tester/RelationalUtilities/PostgreSqlStorageForTesting.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
```

Comment:
It looks fine to me.  If you want to refactor, I'm happy to assist with the PostgreSQL code.